.travis.yml

```diff
@@ -25,6 +27,15 @@ notifications:
 
 before_script:
   - chmod +x gradlew
+  # ARM architecture is used instead of x86 (which is 10x faster) of the lack of support from CI due
+  # to complications of creating a virtual machine within a virtual machine. This may be solved
+  # eventually and would significantly speed some things up. - Chris Lee 4/04/2016
```

Comment:
As long as it's API level 21, it shouldn't matter how the emulator is create. I can make a note in the wiki.